simapp/integration/client/grpc/tmservice/service_test.go

```diff
@@ -27,20 +31,29 @@ type IntegrationTestSuite struct {
 }
 
 func TestIntegrationTestSuite(t *testing.T) {
-	t.Skip() // to be re-enabled in https://github.com/cosmos/cosmos-sdk/pull/12482/
+	//t.Skip() // to be re-enabled in https://github.com/cosmos/cosmos-sdk/pull/12482/
```

Comment:
Re-enabled for now to see if these changes reduce failures.

Reply:
If you can re-enable it that's even better. We had to disable it because gov was migrated to app wiring.

Reply:
Well it's passing here on this branch...